.ci/atime/tests.R

```diff
@@ -13,6 +13,7 @@ for (extra.arg in extra.args.6107){
       tmp_csv = tempfile()
       fwrite(DT, tmp_csv)
```

Comment:
this test is only of `fwrite()`, where the claimed improvement is for `fread()`, am I missing something?

Reply:
yes you missed something.
fwrite is used in setup, but setup is not timed.
the timing happens on expr which uses fread below.

Reply:
:facepalm: thanks :)